### PR TITLE
Feat : Added Card Component

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my card component?

Link - https://water-style-ui.netlify.app/components/card.html

Thanks in advance

### CODE REVIEW

**Review by syed-shaida-hussain — APPROVED** [review]
This looks very clean. No changes needed.

File: Components/alert.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
                 <li><a href="badge.html">Badge</a></li>
                 <li><a href="" class = "active-link">Alert</a></li>
                 <li><a href="button.html">Buttons</a></li>
-                <li><a href="">Image</a></li>
+                <li><a href="card.html">Cards</a></li>
```
Comment (viru2001) [inline Components/alert.html:33]: you should use relative paths. 
you can refer this 
https://stackoverflow.com/questions/24028561/relative-path-in-html

File: Components/card.html [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+          </aside>
+
+          <div class="flex flex-column main gap-m debug">
+            <h1>Card with Badge</h1>
+
+            <div class="card-container flex flex-column gap-m">
+              <div class="img-container badge">
+                <img
+                  src="https://images.unsplash.com/photo-1643313947128-849e4da4fd5b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2142&q=80"
+                />
```
Comment (viru2001) [inline Components/card.html]: you should add alt attribute here

File: Components/card.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+              style="
+                width: 503px;
```
Comment (viru2001) [inline Components/card.html:193]: making `width:100%` will make it responsive  and also try to avoid using inline styles

**Review by viru2001 — CHANGES_REQUESTED** [review]
Everything looks good, I suggested changes at some places

File: Components/card.html (hunk 3 above)
Comment (shlok30) [reply Components/card.html:193]: Understood, thanks

File: Components/card.html (hunk 2 above)
Comment (shlok30) [reply Components/card.html]: Thanks

File: Components/alert.html (hunk 1 above)
Comment (shlok30) [reply Components/alert.html:33]: I'll go through this, thanks

**Pull request merged**